suggester/src/main/java/org/opengrok/suggest/Suggester.java

```diff
@@ -209,11 +211,13 @@ private boolean indexExists(final Path indexDir) throws IOException {
         }
     }
 
-    private void shutdownAndAwaitTermination(final ExecutorService executorService, final String logMessageOnSuccess) {
+    private void shutdownAndAwaitTermination(final ExecutorService executorService, Instant start, final String logMessageOnSuccess) {
         executorService.shutdown();
         try {
             executorService.awaitTermination(awaitTerminationTime.toMillis(), TimeUnit.MILLISECONDS);
-            logger.log(Level.INFO, logMessageOnSuccess);
+            logger.log(Level.INFO, logMessageOnSuccess + " (took {0})",
```

Comment:
Maybe rewrite to avoid another concatenation?
```
logger.log(Level.INFO, "{0} (took {1})", new Object[] {logMessageOnSuccess, DurationFormatUtils...)
```